docs/documents/release-process.md

```diff
@@ -4,4 +4,26 @@
    
    * When deploying a review application a script is run to find the first available static route (review-school-experience[1-20] ) and attach that to the review, via terraform. This enables the DFE Sign In process to work.
 
-1. Merging the PR will deploy the changes to the staging environment and, if successful, the production environment
+2. Merging the PR will deploy the changes to the staging environment and, if successful, the production environment
+
+## Configuring a school profile in a review app or local deployment for testing
+
+1. Ensure you have a login to DfE Sign In (pre-prod) with permission to manage any of the schools listed in `db/data/example_schools.csv`
```

Comment:
Do we have a process for dfe sign in? We could at least point at their contact: https://technical-guidance.education.gov.uk/infrastructure/support/#dfe-sign-in